src/main/java/zadanie1.java

```diff
@@ -0,0 +1,9 @@
+public class zadanie1 {
```

Comment:
Checkstyle zauważył w tej linii błąd. To, co znalazł możesz podejrzeć w pull requeście z zakładce "Files changed". Polecam zainstalować sobie dodatek do intellij, żeby na bieżąco informował. Instrukcję znajdziesz tutaj https://bootcamp.javastart.pl/strona/jakosc-kodu

W tym przypadku to informacja, że nazwa klasy powinna być zapisana dużą literą